lib/xcp_service.ml

```diff
@@ -21,6 +21,7 @@ module StringSet = Set.Make(String)
 *)
 let default_service_name = Filename.basename Sys.argv.(0)
 let config_file = ref (Printf.sprintf "/etc/%s.conf" default_service_name)
+let config_dir = ref (Printf.sprintf "/etc/%s.d/conf" default_service_name)
```

Comment:
The common way to do this seems to be to call the directory '/etc/%s.d' or '/etc/%s.conf.d'.   For example: /etc/yum.repos.d, /etc/sudoers.d, /etc/ld.so.conf.d